docs/source/using/experimental-features.adoc

```diff
@@ -151,3 +151,163 @@ To get the status of the DNS server:
 ----
 $ minishift dns status
 ----
+
+[[local-dns-setup-macos]]
+== Local DNS Set-up for macOS
+
+While in offline mode recent versions of macOS does not send out dns queries, and the process for using the local dns server from {project} is a bit more involved then
```

Comment:
Combine these lines and fix grammatical errors:

```adoc
Recent versions of macOS do not send out DNS queries, and the process for using a local DNS server from {project} is more involved than other operating systems.
```

Reply:
Mentioning `offline` is important here, as this happens only when there is no network connection.